modules/ssp_genieeBidAdapter.md

```diff
@@ -0,0 +1,46 @@
+# Overview
+
+```
+Module Name: Geniee Bid Adapter
```

Comment:
Should this be `Geniee SSP Adapter`?

Reply:
I have used the md that is present in the prebid.js repo. Let me know if I need to change this.